src/components/App.tsx

```diff
@@ -2009,6 +2010,7 @@ class App extends React.Component<AppProps, AppState> {
   ): NonDeleted<ExcalidrawTextElement> | null {
     const element = this.getElementAtPosition(x, y, {
       includeBoundTextElement: true,
+      preferSelected: true,
```

Comment:
not sure if this doesn't break some sticky-note cases

Reply:
it actually does break something: pressing enter on a selected container currently prefers to edit text that's at container center, even if the text is unbound. Unclear what the behavior should be actually, but I'll keep production behavior for now (and add a test for this).